code/__DEFINES/species.dm

```diff
@@ -112,3 +112,31 @@
 		SPECIES_UNATHI, \
 		SPECIES_VAURCA_WORKER, SPECIES_VAURCA_WARRIOR, SPECIES_VAURCA_BULWARK, SPECIES_VAURCA_BREEDER, \
 	)
+
+#define ALL_HUMANS list(\
+		SPECIES_HUMAN, SPECIES_HUMAN_OFFWORLD, SPECIES_HUMAN_VATGROWN, \
+	)
+
+#define ALL_DIONAE list(\
+		SPECIES_DIONA, SPECIES_DIONA_COEUS, \
+	)
+
+#define ALL_SKRELL list(\
+		SPECIES_SKRELL, SPECIES_SKRELL_AXIORI, \
+	)
+
+#define ALL_TAJARA list(\
+		SPECIES_TAJARA, SPECIES_TAJARA_ZHAN, SPECIES_TAJARA_MSAI, SPECIES_TAJARA_TESLA_BODY, \
+	)
+
+#define ALL_VAURCA list(\
+		SPECIES_VAURCA_WORKER, SPECIES_VAURCA_WARRIOR, SPECIES_VAURCA_BREEDER, SPECIES_VAURCA_WARFORM, SPECIES_VAURCA_BULWARK, \
+	)
+
+#define ALL_IPCS list(\
+		SPECIES_IPC, SPECIES_IPC_BISHOP, SPECIES_IPC_G1, SPECIES_IPC_G2, SPECIES_IPC_SHELL, \
+		SPECIES_IPC_UNBRANDED, SPECIES_IPC_UNBRANDED_REMOTE, SPECIES_IPC_XION, SPECIES_IPC_ZENGHU, \
+		SPECIES_IPC_SHELL_ROGUE, SPECIES_IPC_XION_REMOTE, SPECIES_IPC_PURPOSE_HK, \
+	)
```

Comment:
All of these should have names like `ALL_HUMAN_SPECIES` to be a bit more consistent with other defines